src/main/java/uk/nhs/prm/e2etests/model/nems/NemsResolutionMessage.java

```diff
@@ -2,11 +2,25 @@
 
 import lombok.Getter;
 
+import java.util.Objects;
+
 @Getter
 public class NemsResolutionMessage {
     String nemsMessageId;
     String status;
 
+    public boolean hasTheSameContentAs(Object o) {
+        if (this == o) return true;
+        if (o == null) return false;
+        NemsResolutionMessage that = (NemsResolutionMessage) o;
```

Comment:
What kind of variable name is 'that' !?

Reply:
Funnily enough, when you override equals and hashcode - this does seem to be the autogenerated variable name!

Reply:
Addressed